src/GeneralStateTestsFiller/stPreCompiledContracts2/CALLCODEBlake2fFiller.json

```diff
@@ -0,0 +1,418 @@
+            "balance": "999999999999999915052"
+          }
+        }
+      },
+      {
+        "indexes": {
+          "data": [15],
+          "gas": -1,
+          "value": -1
+        },
+        "network": [">=Istanbul"],
+        "result": {
+          "1000000000000000000000000000000000000000": {
+            "storage": {
+              "0x01": "0x01",
+              "0x02": "0xf5b5dbc5e76321b3ef75c8ead211dc1f7e0a0999767ecbbb5daf9507d5a8f87f",
+              "0x03": "0xcdf83e5498ffd974872785043dc19af8567417c800efe05637758ee39fd5e161"
+            }
+          },
+          "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+            "balance": "999999999999999914992"
+          }
+        }
+      },
+      {
+        "indexes": {
+          "data": [16],
+          "gas": -1,
+          "value": -1
+        },
+        "network": [">=Istanbul"],
+        "result": {
+          "1000000000000000000000000000000000000000": {
+            "storage": {
+              "0x01": "0x01",
+              "0x02": "0xb02406865b80cde9fec1e321559891be456158a6d71cf0eac1eb845c70d4a3d8",
+              "0x03": "0x948e82cc975dcbd6984780bab7e236382fb585df5bc4b043cb3e866544413d92"
+            }
+          },
+          "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+            "balance": "999999999999999914944"
+          }
+        }
+      },
+      {
+        "indexes": {
+          "data": [17, 18, 19],
+          "gas": -1,
+          "value": -1
+        },
+        "network": [">=Istanbul"],
+        "result": {
+          "1000000000000000000000000000000000000000": {
+            "storage": {
+              "0x01": "0x01",
+              "0x02": "0xccfc282ed60927145b46f8d0fa97afd07010c51d20821e9748923ea42a37a0fa",
+              "0x03": "0x0609a13be7c1e14b6e10a4b63d85d1d56d3d370d80f97b0a61a4f22ed6462dee"
+            }
+          },
+          "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+            "balance": "999999999999999914920"
+          }
+        }
+      }
+    ],
+    "pre": {
+      "1000000000000000000000000000000000000000": {
+        "balance": "",
+        "code": "{ (CALLDATACOPY 0 0 (CALLDATASIZE)) [[1]] (CALLCODE (GAS) 9 0 0 (CALLDATASIZE) 1000 64) [[2]](MLOAD 1000) [[3]](MLOAD 1032)}",
+        "nonce": "0",
+        "storage": {}
+      },
+      "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+        "balance": "1000000000000000000000",
+        "code": "",
+        "nonce": "0",
+        "storage": {}
+      }
+    },
+    "transaction": {
+      "data": [
+        "0x000000016a09e667f2bd8948bb67ae8584caa73b3c6ef372fe94f82ba54ff53a5f1d36f1510e527fade682d19b05688c2b3e6c1f1f83d9abfb41bd6b5be0cd19137e217907060504030201000f0e0d0c0b0a090817161514131211101f1e1d1c1b1a191827262524232221202f2e2d2c2b2a292837363534333231303f3e3d3c3b3a3938000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000080000000000000000001",
```

Comment:
need a comment of what this bytecode means. you could also use LLL in transaction data

Reply:
It has specific EIP-152 meaning, so I pointed that out in the "// data" comments, and provided a test case by testcase description of what is important in that blob.

These are also the same strings found in the reference implementation, with additions.